doc/en/components/charts/chart-type-overview.md

```diff
@@ -1,32 +1,212 @@
+
+### Sparkline Chart 
+
+A Sparkline Chart, or Sparkline Graph is a type of category graph intended for rendering within a small-scale layout such as within a grid cell, or anywhere a word-sized visualization is needed to tell a data story. Like other $PlatformShort$ chart types, the Sparkline Chart has several visual elements and corresponding features that can be configured and customized such as the chart type, markers, ranges, trendlines, unknown value plotting, and tooltips. Sparkline charts can render as a Line Chart, Area Chart, Column Chart or Win / Loss Chart. The difference between the full-sized chart equivalent to the Spark-chart, is the Y-Axis (left side labels) and X-Axis (bottom labels) are not visible. Learn more about our [sparkline charts](../sparkline.md).
+
+### Bubble chart 
+
+A Bubble Chart, or Bubble Graph, is used to show data comprising of three numeric values. Two of the values are plotted as an intersecting point using a Cartesian (X, Y) coordinate system, and the third value is rendered as the diameter size of the point. This gives the Bubble Chart its name - a visualization of varying sized bubbles along the X and Y coordinates of the plot. The $PlatformShort$ Bubble Chart is used to show relationships of data correlations with the data value differences rendered by size. You can also use a fourth data dimension, typically color, to further differentiate the values in your Bubble chart. Learn more about our [bubble charts](chart-types-scatter.md#$PlatformShort$-Scatter-Bubble-Chart).
+
+### Financial chart / Stock Chart 
+
+The $PlatformShort$ Financial or Stock Chart, is a composite visualization that renders stock data and financial data in a time-series chart that includes interactive visual elements in a toolbar like day / week / month filters, chart type selection, volume type selection, indicators selection and trends lines selection. Designed for customization, the $PlatformShort$ Stock Chart can be customized in any way to give an easier visualization and interpretation of your data. The financial chart renders the date-time data along the X-Axis (bottom labels) and shows fields like Open, High, Low and Close volumes. The type of chart to render the Time-Series data can be Bar, Candle, Column, or Line. Learn more about our [financial charts](chart-types-financial.md).
+
+### Column chart 
+
+The $PlatformShort$ Column Chart, or Column Graph is among the most common category chart types used to quickly compare frequency, count, total, or average of data in different categories with data encoded by vertical bars of equal width and differing lengths. They are ideal for showing variations in the value of an item over time, data distribution, sorted data ranking (high to low, worst to best). Data is represented using a collection of rectangles that extend from the top to bottom of the chart towards the values of data points. Learn more about our [column charts](chart-types-column.md).
+
```

Comment:
https://share.goabstract.com/1b2f1572-bee7-4a6f-987b-3a5067e8a0f8?mode=design&sha=cbb507583316514fd9d47f72830350341a36dbc0

[annual-change-primary-energy-source.xlsx](https://github.com/IgniteUI/igniteui-xplat-docs/files/5886394/annual-change-primary-energy-source.xlsx)

Reply:
ok, we are creating this sample